File: Safari-611.1.10/LayoutTests/http/tests/eventsource/resources/es-eof.php
<?php
header("Content-Type: text/event-stream");

$id = floatval($_SERVER['HTTP_LAST_EVENT_ID']) + 1;

echo "retry: 0\n";
echo "id: " . $id . "\n";
echo "data: DATA" . intval($id) . "\n\n";

echo "id: " . ($id + 0.1) . "\n";
echo "event: msg\n";
echo "data: DATA" . str_repeat("\n", $id - 1);
?>

File: LayoutTests/http/tests/eventsource/resources/es-eof.py
#!/usr/bin/env python3

import os
import sys

id = float(os.environ.get('HTTP_LAST_EVENT_ID', 0)) + 1

sys.stdout.write(
    'Content-Type: text/event-stream\r\n\r\n'
    'retry: 0\n'
    'id: {id}\n'
    'data: DATA{data}\n\n'
    'id: {id_inc}\n'
    'event: msg\n'
    'data: DATA{newlines}'.format(id=id, data=int(id), id_inc=id + 0.1, newlines='\n' * (int(id) - 1))
)